src/Cli/dotnet/Parser.cs

```diff
@@ -93,6 +93,8 @@ private static Command ConfigureCommandLine(Command rootCommand)
             .UseHelp()
             .UseHelpBuilder(context => new DotnetHelpBuilder(context.Console))
             .UseValidationMessages(new CommandLineValidationMessages())
+            .UseParseDirective()
```

Comment:
From what I've read, this directive (if present), shows how the commandline arguments were parsed, but it never executes the actual command. What's the benefit in having this? I can see it being useful when testing an application, or is this to ensure that any dotnet command would implicitly support [parse]?

Reply:
This is one of the opt in features that we get for free from System.CommandLine, so with this change all dotnet commands will implicitly support [parse].